en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+# Managing Common API Policies
+
+## Add common API Policies in an environment
```

Comment:
Why would someone add a common API policy? Can we expand on the use case aspect here?

Reply:
I think this is already mentioned in the docs. Refer https://apim.docs.wso2.com/en/latest/design/api-policies/create-policy/

Reply:
Let's add a link to this please.

Reply:
+1

Reply:
Fixed by [021c130](https://github.com/wso2/docs-apim/pull/6103/commits/021c13019ef5f9d516c36b1f4dd9088bf80d542e)